libs/snorenotify/BUILD

```diff
@@ -0,0 +1,5 @@
+source /etc/profile.d/qt5.rc &&
+
+default_cmake_build &&
+
+OPTS+="-DWITH_QT4=OFF"
```

Comment:
Change this line to;
OPTS+=" -DWITH_QT4=OFF"

Note the space between the double quote and dash.